packages/components/src/scroll-list/scroll-list.tsx

```diff
@@ -171,18 +162,16 @@ export function ScrollList<T, EL extends HTMLElement = HTMLDivElement>({
     const scrollListRef = (scrollListRoot?.props?.ref as React.RefObject<HTMLElement>) || defaultScrollListRef;
     const defaultListRef = useRef<HTMLElement>(null);
     const listRef = (listRoot?.props?.ref as React.RefObject<HTMLDivElement>) || defaultListRef;
-    /**
-     * get scroll position of scrollWindow and trigger re-rendering on its change
-     */
+    /** get scrollWindow's or Window's scroll position and trigger re-rendering on its change */
     const scrollPosition = useScroll({
         isHorizontal,
         ref: scrollWindow,
     });
+    /** get scrollWindow's or Window's size and trigger re-rendering on its change */
```

Comment:
don't you think that this comment is kind of obvious if you know how react hooks work ?

Reply:
at lease second part about rerendering

Reply:
Thought about it as more of reminder of scrollWindow/Window conundrum 🤷🏼